content/rule_book/material_and_questions.md

```diff
@@ -1,10 +1,33 @@
 # Material and Questions
 
-**Version**
-: The material version is the New International Version (NIV 2011) of the Bible.
+Bible Quizzing meets consist of a series of quizzes, which themselves consist of a series of questions. Questions are written based on a quiz season's material. Material refers to the Scriptural scope of content set for each season. The Bible edition used is the *New International Version* (NIV) 2011 edition of the Bible. There is an 8-season rotation of material. 4 are defined as "Narrative" and 4 are defined as "Epistle" in terms of a material's style.
 
-**Material**
-: The biblical content that is the basis for competition for the quiz year. This includes Book Name, Chapter Number, and Verse Number. This does not include any specifically excluded material. Of the eight years of material, four are defined as Narrative, and four are defined as Epistle.
+**Question Type**
+: Every quiz question is 1 of a particular set of question types, such as *Chapter Verse Reference* (CVR) and *Finish the Verse* (FTV).
+
+**Question Type Group**
+: Question types are part of 1 or more question type groups, such as Reference and Finish.
+
+**Question Text**
+: Every word required to ask a question
+
+**Answer Text**
+: Every word in addition to the question text required to answer a question perfectly
```

Comment:
I don't like the use of the word "perfectly" here.  We don't want to imply any notion of "word-perfect" exists for each "answer text."

Reply:
Sure, but that's not what this does. "Answer Text" is the exact text if the question was answered perfectly. It's explicitly not required for INT, MA, etc.

Happy to hear alternate phraseology suggestions, of course.

Reply:
Oh right. Hmmmmm.  "Every verbatim material of the written answer, in addition to the question text" ??? I don't feel great about that. But the "required to answer a question perfectly" is misleading.

Reply:
I don't see how it's misleading. We're not defining the "answer"/"response". We're defining the "answer text". And we define "answer"/"response" explicitly and differently.

Reply:
yeah, you're right, because "required to answer a question perfectly" != "required to be correct"